FizzBuzz 1/FizzBuzz/Program.cs

```diff
@@ -1,42 +1,117 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace FizzBuzz
 {
     class Program
     {
+        static Boolean ReadYN(string message)
+        {
+            char answer = 'a';
+            var valid = new List<char>();
+            valid.Add('y');
+            valid.Add('n');
+            while (!valid.Contains(answer))
+            {
+                Console.Write(message);
+                try
+                {
+                    answer = Char.Parse(Console.ReadLine());
+                    answer = Char.ToLower(answer);
```

Comment:
I'd probably just do this as one statement:
```
answer = Char.ToLower(
  Char.Parse(Console.ReadLine())
)
```
Otherwise I find it a bit confusing that we're assigning to `answer` twice 🙂 ]